projects/plugins/jetpack/_inc/client/recommendations/style.scss

```diff
@@ -13,6 +13,7 @@
 
 	@include breakpoint( '<660px' ) {
 		grid-template-columns: 100% 0%;
+		background: #fff;
```

Comment:
Another solution would have been to apply the gradient to viewports with a width of 660px and above, but the codebase seems to favour a desktop first approach: we wouldn't have been able to use the `breakpoint` mixin as is.